### PR TITLE
Design version 0.0.2

### PR DESCRIPTION
**Pull request by humanamburu**
image

### CODE REVIEW

**Comment by kirilknysh** [comment]
v2

image

**Comment by humanamburu** [comment]
image

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+node_modules
+npm-debug.log
+yarn-error.log
+dist
+dist-prod
+.idea/
```
Comment (Shastel) [inline .gitignore]: missing newline 😄

File: index.html [hunk 2]
```diff
@@ -1,72 +1,50 @@
...
+                        <path d="M192.440223,144.644612 L224.220111,144.644612 L224.220111,68.3393384 L188.415329,32.5345562 L165.943007,55.0068785 L192.440223,81.5040943 L192.440223,144.644612 L192.440223,144.644612 Z M224.303963,160.576482 L178.017688,160.576482 L113.451687,160.576482 L86.954471,134.079266 L98.1906322,122.843105 L120.075991,144.728464 L165.104487,144.728464 L120.746806,100.286931 L132.06682,88.9669178 L176.4245,133.324599 L176.4245,88.2961022 L154.622994,66.4945955 L165.775303,55.3422863 L110.684573,0 L56.3485097,0 L56.3485097,0 L0,0 L31.6960367,31.6960367 L31.6960367,31.7798886 L31.8637406,31.7798886 L97.4359646,31.7798886 L120.662954,55.0068785 L86.7029152,88.9669178 L63.4759253,65.7399279 L63.4759253,47.7117589 L31.6960367,47.7117589 L31.6960367,78.9046839 L86.7029152,133.911562 L64.3144448,156.300033 L100.119227,192.104815 L154.45529,192.104815 L256,192.104815 L256,192.104815 L224.303963,160.576482 L224.303963,160.576482 Z"/>
+                    </g>
+                </svg>
+            </div>
+            <a href="" class="product-name">WebSocket Challenge</a>
+        </li>
+    </ul>
+</main>
 </body>
 </html>
```
Comment (Shastel) [inline index.html]: newline

**Pull request closed without merging**